### ISSUE TITLE
xfce panel is flickering as soon as you klick left or right on the tray icon of MSM

### ISSUE THREAD

**Issue by philmmanjaro** [manjaro/manjaro-settings-manager#63]
This might be related to https://github.com/manjaro/manjaro-settings-manager/commit/26f190fcee2625236231f4b4281169fd91cdcb8d

![tray-issue-xfce](https://cloud.githubusercontent.com/assets/5050022/14894803/c2076b42-0d75-11e6-9c95-88c959ea3a1e.png)

**Comment by philmmanjaro** [comment]
I've to correct myself. If you have the setting intelegent autohide panel enabled somehow by clicking on the tray icon the panel gets triggered to hide itself. However it gets mad on it. If you disable that function it acts normal. Now we have to find out why the new tray icon triggers that XFCE feature now.

**Comment by Kirek** [comment]
Maybe it doesn't like svg files... I will check if a png works.

**Comment by Kirek** [comment]
Can you check with 0.4.96?

**Comment by philmmanjaro** [comment]
I'll check when I'm home ...

**Comment by philmmanjaro** [comment]
With the latest version I see no tray icon at all. However the process runs:
```
[phil@manjaro Arbeitsfläche]$ ps -ef | grep msm
phil       987   806  0 00:06 ?        00:00:00 msm_notifier
phil      1145  1134  0 00:07 pts/0    00:00:00 grep --colour=auto msm
```

**Comment by Kirek** [comment]
there is no notification icon until something happens, this is like the old notifier...

**Comment by philmmanjaro** [comment]
Ok, however it still flickers ...

**Comment by philmmanjaro** [comment]
Seems to be the same was already posted: https://github.com/manjaro/manjaro-settings-manager/issues/50

**Comment by philmmanjaro** [comment]
The flickering continues also with the new notifiers. It has nothing to do if the icon is png, jpg or svg. It was also in our 0.4.x release already present. However, as soon as you click on options or any other menu entry the flickering stops. Seems it flickers as long as the menu is displayed. weird.

**Comment by petsam** [comment]
An update:
I cannot find the msm-notifier anywhere in my system. Is it some temp-fix?.
Nevertheless, the same issue happens to me with the Viber notification icon in the tray area.

**Comment by scachemaille** [comment]
@petsam  you mean you don't see any icon? or the process is not running?
if the process is running you won't see icons (for no kde notifier) until you get a notification for available kernel or language pack.
you can change the notication setting with manjaro notifier settings. should be in settings part of the menu

**Comment by petsam** [comment]
Disregard the disappearance, it was just not installed after the split of msm and msm-notifier. For the flickering it seems to have some connection with Qt. I opened a topic at forum.

**Comment by philmmanjaro** [comment]
Seems to be fixed within **v17.0**.
